### PR TITLE
Remove node-static

### PR DESCRIPTION
**Pull request by vrunoa**
In regards to the statement published by npm: 
```
All versions of node-static are vulnerable to a Denial of Service. The package fails to catch an exception when user input includes null bytes. This allows attackers to access http://host/%00 and crash the server.
Remediation No fix is currently available. Consider using an alternative package until a fix is made available.
```
https://www.npmjs.com/advisories/1208

* Changing the static server approach to use express instead of node-static,
* Added circle-ci stage running storybook static test,

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -21,3 +21,33 @@ jobs:
...
+  e2e-test:
+    docker:
+      - image: saucelabs/storybook-base:alpha-rc.0.0.1
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "package.json" }}
```
Comment (eugenekatsov-sauce) [inline .circleci/config.yml:33]: Should this be indented?

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+## 0.20.0
+- Remove node-static dependency due to security issues. [https://www.npmjs.com/advisories/1208](https://www.npmjs.com/advisories/1208)
```
Comment (eugenekatsov-sauce) [inline CHANGELOG.md:4]: The url starts with [ and ends with )

**Comment by loyalchow** [comment]
@vrunoa looks good 👍 - but do you know why the circleci build is failing?

File: .circleci/config.yml (hunk 1 above)
Comment (vrunoa) [reply .circleci/config.yml]: Fixed

**Comment by vrunoa** [comment]
@loyalchow renamed the cache keys, it's working again.

**Pull request merged**